views/site/partials/tour/2.php

```diff
@@ -3,7 +3,7 @@
         <div class="col-md-7">
             <img src="<?= Yii::getAlias('@web/image/tour/2.png') ?>"
                  alt="Welcome Page"
-                 srcset="<?= Yii::getAlias('@web/image/tour/2@2x.png') ?>"
+                 srcset="<?= Yii::getAlias('@web/image/tour/2.png 1x, @web/image/tour/2@2x.png 2x') ?>"
```

Comment:
It won't work either - `getAlias()` will resolve only first URL.

Reply:
I'll fix it.

Reply:
https://github.com/yiisoft-contrib/yiiframework.com/commit/456e5371a1744bdcd21881333aa154937f15964e